### PR TITLE
Extra details for environment pages.

### LINKED ISSUE #13: Mainnet page
Some more detail on the mainnet page would be nice. Quoting a comment from @s-ben 

> insert general advice about mainnet a dev would need. E.g. warnings about deploying to mainnet, what stage you need to be in to deploy to mainnet, any caveats/changes from simnet/testnet to think about

### PR DESCRIPTION
**Pull request by jholdstock**
Closes #13

### CODE REVIEW

File: docs/environments/testnet.md [hunk 1]
```diff
@@ -9,12 +9,24 @@ test new applications and experiment without risk before moving onto mainnet.
 Decred is currently on its 3rd testnet, also known as testnet3. Testnets are
 periodically reset to help keep a manageable blockchain file size.
 
+Special rules apply to testnet in order to make it easier to work with.
+For example, if no blocks are found on testnet in a 10 minute period, the PoW
+difficulty will drop so that a block can be generated using the dcrd `generate`
```
Comment (s-ben) [inline docs/environments/testnet.md]: The formatting on `generate` is slightly jarring...Also, the example command isn't introduced in text. This is OK, but would suggest something like:

...can be generated using dcrd's generate command, as shown below:

```no-highlight
dcrctl --testnet generate 1
```

File: docs/environments/simnet.md [hunk 2]
```diff
@@ -1,11 +1,16 @@
-# <img class="dcr-icon" src="/img/dcr-icons/Dcrtl.svg" /> Simnet 
+# <img class="dcr-icon" src="/img/dcr-icons/Dcrtl.svg" /> Simnet
 
 ---
 
 When developing Decred applications or testing potential changes, it is often extremely useful to have a test network where difficulty levels are low enough to generate blocks as needed.
 In order to facilitate these scenarios, dcrd provides a simulation network (`--simnet`), where the difficulty starts extremely low to enable fast CPU mining of blocks.
 Simnet also has some modified functionality that helps developers avoid common issues early in development.
 
+As simnet is a private, temporary network for personal use, there are no public
```
Comment (s-ben) [inline docs/environments/simnet.md]: >As simnet is a private, temporary network for personal use, there are no public...

Is this comma legal? I know replacing periods with commas is the new trend, and I'm starting to do this myself, but it still feels a little wrong :/

Comment (jholdstock) [reply docs/environments/simnet.md]: Reworded these:

> As simnet is a temporary network for private use, there are no public block explorers or faucets available.

File: docs/environments/testnet.md (hunk 1 above)
Comment (jholdstock) [reply docs/environments/testnet.md]: Implemented this suggestion

**Comment by s-ben** [comment]
@jholdstock lgtm

**Pull request merged**